static/src/stylesheets/module/_chapters.scss

```diff
@@ -1,19 +1,85 @@
 .chapters {
-    background-color: colour(neutral-8);
-    border-top: 3px solid colour(news-main-2);
-    margin-left: 0;
-    padding: $gs-baseline/2 $gs-gutter/2;
+    position: relative;
     margin-bottom: $gs-baseline;
 
-    ol {
-        margin: $gs-baseline/3 0 0;
+    @include mq($until: desktop) {
+        margin-top: -($gs-baseline / 1.5);
+        border-bottom: 1px dotted colour(neutral-6);
+    }
+
+    .content__main-column & {
+        @include mq(desktop) {
+            display: none;
+        }
 
-        @include mq(tablet) {
-            column-count: 2;
+        .is-not-modern & {
```

Comment:
I was imagining you'd just add modern-visible to  <button class="chapters__button js-toggle-chapters meta-button hide-on-desktop" data-link-name="weather-toggle-chapters">

as it functionally it works without JavaScript apart from the toggle?